client/src/pages/CreateProfilePage/CreateProfilePage.css

```diff
@@ -0,0 +1,44 @@
+.create-profile-wrapper-div {
+  display: flexbox;
```

Comment:
`display: flexbox` isn't a thing! change to `display: flex` if you're trying to use flexbox. it might affect how the rest of the page looks like since the rest of the css is assuming flexbox isn't being applied